impl/src/main/java/org/jboss/weld/bootstrap/BeanDeploymentModule.java

```diff
@@ -55,8 +55,13 @@
     private final boolean web;
     private final ObserverNotifier notifier;
     private final Set<BeanManagerImpl> managers;
+    private final boolean fallbackModule;
```

Comment:
I don't think we need a special flag to mark a fallback - the default/fallback BDM has a specific `id`.